vimeo-networking/src/main/java/com/vimeo/networking/model/VideoBadge.java

```diff
@@ -0,0 +1,91 @@
+package com.vimeo.networking.model;
+
+import com.vimeo.stag.GsonAdapterKey;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+
+/**
+ * A model representing a "badge" that a video can display. The badge represents
+ * an award, and is usually displayed over the thumbnail of the video, or near
+ * video data (name, description, etc)
+ * Created by zetterstromk on 10/4/16.
+ */
+public class VideoBadge implements Serializable {
+
+    private static final long serialVersionUID = -5343389171512787927L;
+
+    @Nullable
+    @GsonAdapterKey("type")
+    public String mType;
+
+    @Nullable
+    @GsonAdapterKey("festival")
+    public String mFestival;
+
+    @Nullable
+    @GsonAdapterKey("link")
+    public String mLink;
+
+    @Nullable
+    @GsonAdapterKey("text")
+    public String mText;
+
+    @Nullable
+    @GsonAdapterKey("pictures")
+    public PictureCollection mPictureCollection;
+
+    // -----------------------------------------------------------------------------------------------------
+    // Getters
+    // -----------------------------------------------------------------------------------------------------
+    // <editor-fold desc="Getters">
+
+    /**
+     * A type of the badge, such as "staffpick", "vod", or "weekendchallenge". These are
```

Comment:
It may not be used, however, since this is for the public API, I think we should add all of the available fields.